packages/demo/src/factories/firebaseUpdateItem.js

```diff
@@ -0,0 +1,7 @@
+export default function firebaseUpdateItem ({firebase, moduleName, uid, id, payload}) {
+  const payloadWithDates = Object.assign({}, {
+    updated_at: {'.sv': 'timestamp'},
+    created_at: payload.created_at || {'.sv': 'timestamp'}
+  }, payload)
+  return firebase.update(`${moduleName}.${uid}.${id}`, payloadWithDates)
+}
```

Comment:
Maybe we should ask @christianalfoni if these helpers could be migrated inside the firebase provider ?